src/ts-ast.ts

```diff
@@ -384,6 +386,29 @@ export class Property {
   }
 }
 
+export class ConstValue {
+  readonly kind = 'constValue';
+  name: string;
```

Comment:
`readonly` for these other fields too?

Reply:
readonly is on the kind just so that I can use it as a type guard. Not sure it's important to lock in the other fields.